src/overlays/actors/ovl_En_Ds2n/z_en_ds2n.c

```diff
@@ -1,7 +1,7 @@
 /*
  * File: z_en_ds2n.c
  * Overlay: ovl_En_Ds2n
- * Description:
+ * Description: Medicine Shop Owner from OoT (unused)
```

Comment:
```suggestion
 * Description: Potion Shop Owner from OoT (unused)
```
I think it's called the Potion Shop, not the Medicine Shop

Reply:
In the The Legend of Zelda: Encyclopedia, he's called "Medicine Shop Owner". However, you are totally correct in that the shop is called the "Potion Shop". So even the supplementary material isn't consistent. I'll switch it to "Potion Shop," since that's the in-game name of it.